wordle-web/components/settings-dialog.vue

```diff
@@ -79,9 +80,34 @@ export default class SettingsDialog extends Vue {
       error: colors.pink.accent3,
       success: colors.deepPurple.lighten4,
     }
-
     this.$vuetify.theme.themes.dark = purpleTheme
     this.$vuetify.theme.themes.light = purpleTheme
   }
+  fallTheme() {
+    const fallTheme = {
+      primary: colors.brown,
+      accent: colors.deepOrange.accent3,
+      secondary: colors.deepOrange,
+      info: '#FFEB3B',
+      warning: colors.orange.accent1,
+      error: colors.orange.accent3,
+      success: colors.brown.lighten4,
+    }
+    this.$vuetify.theme.themes.dark = fallTheme
+    this.$vuetify.theme.themes.light = fallTheme
```

Comment:
You may consider wrapping this into its own function called something like 'setTheme' that takes in that object and sets it. That way you aren't writing this twice. Not a big deal though. Good job here.